ykcapi/src/lib.rs

```diff
@@ -58,13 +58,14 @@ pub extern "C" fn __ykrt_control_point(
     ctrlp_vars: *mut c_void,
     // Frame address of caller.
     frameaddr: *mut c_void,
+    smid: u64,
```

Comment:
Let's document this, because I don't think I'll remember it.

I think it's also worth documenting (at least somewhere in yk) that ykllvm explicitly writes out a `u64`. I wrote a long comment suggesting this should be `size_t` before realising that ykllvm really does write out a `u64` :)

Reply:
Yeah, stackmap ids are `u64`s. :-/